app/models/recipe.rb

```diff
@@ -0,0 +1,10 @@
+class Recipe < ApplicationRecord
+  validates :title, presence: true
+  validates :description, presence: true, on: :update
```

Comment:
現在、画像とタイトルを入力した後に一度saveメソッドで保存をし、できたものをupdateメソッドで編集していくという仕様になっています。そのため、updateメソッドの時にdescriptionを入力するので、on:を使ってエラーが起こるのを防ぎました。

Reply:
口頭で説明しましたが、登録画面でdescriptionも入力できるようにしてください。
```suggestion
  validates :description, presence: true
```